launcher/game/EasyDialogsWin.py

```diff
@@ -607,7 +607,7 @@ def AskFileForOpen(
     ofn.lpstrTitle = windowTitle
     ofn.lpstrInitialDir = defaultLocation
 
-    if typeList and filter(None, typeList):
+    if typeList and [t for t in typeList if t]:

```

Comment:
This would probably be more straightforward and easier to understand as:
```python
    if typeList and any(typeList):
```

Reply:
@Uzume Thanks! Fixed.